Explorer/Assets/DCL/InWorldCamera/PhotoDetail/PhotoDetailPoolManager.cs

```diff
@@ -0,0 +1,121 @@
+                actionOnGet: equippedWearable => equippedWearable.view.gameObject.SetActive(false),
+                actionOnRelease: equippedWearable => EquippedWearableRelease(equippedWearable, unusedEquippedWearablePoolObjectParent),
+                actionOnDestroy: equippedWearable => GameObject.Destroy(equippedWearable.view.gameObject),
+                collectionCheck: true,
+                equippedWearableDefaultCapacity,
+                equippedWearableMaxSize);
+        }
+
+        private void EquippedWearableRelease(EquippedWearableController equippedWearable, GameObject unusedEquippedWearablePoolObjectParent)
+        {
+            equippedWearable.view.transform.SetParent(unusedEquippedWearablePoolObjectParent.transform, false);
+            equippedWearable.view.gameObject.SetActive(false);
+        }
+
+        private EquippedWearableController CreateEquippedWearable(
+            EquippedWearableView equippedWearablePrefab,
+            IWebBrowser webBrowser,
+            IDecentralandUrlsSource decentralandUrlsSource,
+            IThumbnailProvider thumbnailProvider,
+            NftTypeIconSO rarityBackgrounds,
+            NFTColorsSO rarityColors,
+            NftTypeIconSO categoryIcons)
+        {
+            EquippedWearableView view = GameObject.Instantiate(equippedWearablePrefab);
+            return new EquippedWearableController(view, webBrowser, decentralandUrlsSource, thumbnailProvider, rarityBackgrounds, rarityColors, categoryIcons);
+        }
+
+        private void VisiblePersonRelease(VisiblePersonController visiblePerson, GameObject unusedVisiblePersonPoolObjectParent, Sprite emptyProfileImage)
+        {
+            visiblePerson.view.transform.SetParent(unusedVisiblePersonPoolObjectParent.transform, false);
+            visiblePerson.view.profileImage.SetImage(emptyProfileImage);
+            visiblePerson.view.gameObject.SetActive(false);
+        }
+
+        private VisiblePersonController CreateVisiblePerson(
+            VisiblePersonView visiblePersonPrefab,
+            IWebRequestController webRequestController,
+            IProfileRepository profileRepository,
+            IMVCManager mvcManager,
+            IWearableStorage wearableStorage,
+            IWearablesProvider wearablesProvider,
+            ChatEntryConfigurationSO chatEntryConfiguration)
+        {
+            VisiblePersonView view = GameObject.Instantiate(visiblePersonPrefab);
+            return new VisiblePersonController(view, webRequestController, profileRepository, mvcManager, wearableStorage, wearablesProvider, this, chatEntryConfiguration);
+        }
+
+        public VisiblePersonController GetVisiblePerson(RectTransform parent)
+        {
+            VisiblePersonController controller = visiblePersonPool.Get();
+            controller.view.transform.SetParent(parent, false);
```

Comment:
Is it worth to reparent? 

- It can bring unnecessary overhead calculations while rebuilding the UI
- You have to maintain extra code for the unused gameobjects.

I would just leave them as child, since that's the only place they can be; unlike other pools